File: 2292.c
#include<stdio.h>

int main(){
    int cellNumber = 0;
    int cellAmount = 1;
    int level = 0;
    scanf("%d", &cellNumber);
    
    if(cellNumber == 1)
        level = 1;
    else
        for(level; cellAmount < cellNumber; level++)
            cellAmount = cellAmount + 6 * level;
    
    printf("%d", level);
    
    return 0;
}

File: 2292.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int cellNumber = sc.nextInt();
        int cellAmount = 1;
        int level = 0;
    
        if(cellNumber == 1)
            level = 1;
        else
            for(; cellAmount < cellNumber; level++)
                cellAmount = cellAmount + 6 * level;
        
        System.out.print(level);
    }
}
